packages/manager-react-components/src/components/content/ManagerTile/manager-tile-item.component.tsx

```diff
@@ -1,20 +1,27 @@
+import React, { PropsWithChildren } from 'react';
 import { ODS_TEXT_PRESET } from '@ovhcloud/ods-components';
 import { OdsText } from '@ovhcloud/ods-components/react';
-import React from 'react';
+import { HelpIconWithTooltip } from './help-icon-with-tooltip';
 
-export const ManagerTileItem = ({ children }: React.PropsWithChildren) => {
+export const ManagerTileItem = ({ children }: PropsWithChildren) => {
   return <dl className="flex flex-col gap-1 m-0">{children}</dl>;
 };
 
-const ManagerTileItemLabel = ({ children }: React.PropsWithChildren) => {
+type ManagerTileItemLabelProps = PropsWithChildren<{ labelTooltip?: string }>;
+
+const ManagerTileItemLabel = ({
+  children,
+  labelTooltip,
+}: ManagerTileItemLabelProps) => {
   return (
-    <dt>
+    <dt className="flex items-center gap-2">
       <OdsText preset={ODS_TEXT_PRESET.heading6}>{children}</OdsText>
+      {labelTooltip && <HelpIconWithTooltip labelTooltip={labelTooltip} />}
```

Comment:
Since label is already present, `label` is redundant in `labelTooltip`

Reply:
I guess you mean to call the prop `tooltip` instead of `labelTooltip`, i'm not sure.